views/admin/users.erb

```diff
@@ -16,20 +16,31 @@
             <label>Find users by:</label>
             <br>
             <a href="/admin/users/" class="dcf-btn dcf-btn-secondary">Clear</a>
+            <div>
+                <label for="users-per-page">Users Per Page:</label>
+                <select id="users-per-page" name="users_per_page" class="dcf-input-select">
+                    <% for x in [1, 5, 10] do %>
```

Comment:
The admins might want to display more than just 10 users, we can bring this up during mini demo.

Reply:
I believe when we were gathering requirements they said they wanted to be able to view pages of 1, 5, or 10 users, but I agree with you I think it would make sense for them to want to be able to view more. For the sake of demoing everything on stable I am going to merge this, but I expect them to correct us on this during the demo.